src/NServiceBus.Router/SendOnlyInterface.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Router;
+using NServiceBus.Configuration.AdvancedExtensibility;
+using NServiceBus.Raw;
+using NServiceBus.Transport;
+
+class SendOnlyInterface<T> : Interface where T : TransportDefinition, new()
+{
+    public SendOnlyInterface(string endpointName, string interfaceName, Action<TransportExtensions<T>> transportCustomization, Func<IRawEndpoint, IRuleCreationContext> ruleCreationContextFactory)
+    {
+        this.ruleCreationContextFactory = ruleCreationContextFactory;
+        Name = interfaceName;
+        rawConfig = new SendOnlyRawEndpointConfig<T>(endpointName, ext =>
```

Comment:
Can `RawEndpointConfiguration.CreateSendOnly` be used here directly?

Reply:
Probably! I'll look into it.

Reply:
https://github.com/SzymonPobiega/NServiceBus.Router/pull/24/commits/ed306de90a9b9074a9123164d617c077a1b1d3eb